RInAppMessaging/Classes/Extensions/Int32+IAM.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Int32 {
+    /// Exponential backoff
+    mutating func increaseBackOff() {
+        self = multipliedReportingOverflow(by: 2).partialValue
+    }
+
+    /// Exponential backoff with a random value
+    mutating func increaseRandomizedBackoff(min: Int32 = Constants.Retry.TooManyRequestsError.backOffLowerBoundMS,
+                                            max: Int32 = Constants.Retry.TooManyRequestsError.backOffUpperBoundMS) {
+        increaseBackOff()
+        self += Int32.random(in: min..<max)
```

Comment:
Hmm. I'm looking at the specs. Shouldn't that be [1...60] seconds i.e. `min...max`?

Reply:
`retryDelayMS.increaseRandomizedBackoff()`
retryDelayMS is in millisecond.
But better if random is: `Int32.random(in: 1..<60)*1000`

Reply:
Yeah, I'm not talking about the unit. I mean should we include upper bound in the range?
To change `min..<max` to `min...max`

Reply:
Yes I agree. Fixed.